HospitalFrontend/src/views/CreateSurveyResponse.vue

```diff
@@ -0,0 +1,161 @@
+    <h1>Patient Survey</h1>
+
+    <h2>How satisfied or dissatisfied are you with the following: </h2>
+
+    <div v-if="survey" id="surveysectionlist" class="container w-50">
+       <div class="bg-light rounded" v-for="surveySection in survey.surveySections" v-bind:key="surveySection.Id" v-bind:surveySection="surveySection" >
+            <div class="list-group my-4">
+                <div class="bg-secondary col d-flex justify-content-left rounded-top">
+                    <h5 class="text-left text-white m-1">{{surveySection.sectionName}}</h5>
+                </div>
+                <div class="mb-3" v-for="surveyQuestion in surveySection.surveyQuestions" v-bind:key="surveyQuestion.id">
+                    <div class="my-1"> {{surveyQuestion.question}} </div>
+                    <div class="form-check form-check-inline">
+                        <input type="radio" class="form-check-input ml-2" v-bind:name="'group' + surveyQuestion.id" 
+                            v-on:click="rateQuestion(surveySection, surveyQuestion.id, 1)">
+                        <label class="form-check-label">1</label>
+                        <input type="radio" class="form-check-input ml-2" v-bind:name="'group' + surveyQuestion.id"
+                            v-on:click="rateQuestion(surveySection, surveyQuestion.id, 2)">
+                        <label class="form-check-label">2</label>
+                        <input type="radio" class="form-check-input ml-2" v-bind:name="'group' + surveyQuestion.id"
+                            v-on:click="rateQuestion(surveySection, surveyQuestion.id, 3)">
+                        <label class="form-check-label">3</label>
+                        <input type="radio" class="form-check-input ml-2" v-bind:name="'group' + surveyQuestion.id"
+                            v-on:click="rateQuestion(surveySection, surveyQuestion.id, 4)">
+                        <label class="form-check-label">4</label>
+                        <input type="radio" class="form-check-input ml-2" v-bind:name="'group' + surveyQuestion.id"
+                            v-on:click="rateQuestion(surveySection, surveyQuestion.id, 5)">
+                        <label class="form-check-label">5</label>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <button type="button" class="btn btn-success btn-lg mb-5" v-on:click="saveSurveyResponse">Submit survey</button>
+    </div>
+
+    
+</template>
+
+<script>
+
+import axios from 'axios'
+import api from '../constant/api.js'
+import Toastify from 'toastify-js'
+
+
+
+export default {
+
+    name:'CreateSurveyResponse',
+    data:function() {
+        return{          
+            survey: null,
+            surveyResponse: {
+                surveyId:null,
+                ratedSurveySections:[],
+                doctorSurveySection:null,
+                patientAccountId:1
```

Comment:
Why is patientAccountId initialized to 1

Reply:
Right now there is no login functionality for users. This id will be fetched from session once we implement login.